api/services/NFTServices.js

```diff
@@ -102,21 +102,26 @@ class NFTServices {
 								);
 								const values = namedKeyValue.unwrap().value();
 								return {
-									[namedKey]: values.map((value) =>
-										this.getAttributeConfig(
-											infoNamedKeysConf[namedKey].attributes,
-											value[0].data,
-											value[1].data,
+									[namedKey]:
+										Array.isArray(values) &&
```

Comment:
Just confirm that [namedKey] should be either array or boolean (false) rite?

Reply:
I'm going to update to return empty array for consistency